### PR TITLE
[Triton] enable client.py to run without GPU

### PR DESCRIPTION
**Pull request by Bin-NV**

### CODE REVIEW

File: triton/client.py [hunk 1]
```diff
@@ -32,6 +32,11 @@
 from nvjpeg import NvJpeg
```
Comment (Tyler-D) [inline triton/client.py:32]: Could we move this import if `has_gpu=True`?

File: triton/build_engine.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
+
+#!/bin/bash
+
```
Comment (Tyler-D) [inline triton/build_engine.sh:3]: The root cause of SA's issue should be related to wrong default runtime of docker when [building the image](https://stackoverflow.com/questions/59691207/docker-build-with-nvidia-runtime). Will it be better to keep one-click script and teach customer how to access GPU in the building time ?

Comment (Bin-NV) [reply triton/build_engine.sh:3]: Previously I used docker version is `24.0.6` and set the default docker runtime to nvidia docker but it didn't help. Now with `DOCKER_BUILDKIT=0 ` in the blog your shared, the docker build works smoothly.

docker `20.10` doesn't have this issues with the nvidia docker as default runtime.

However, there is a warning that the `DOCKER_BUILDKIT=0 ` will be DEPRECATED, so we may face the same error in future with latest docker version, I suggestion we keep this build_engine.sh for users who are using latest version of docker and haven't set the nvidia docker as default runtime.

how do you think?

Comment (Tyler-D) [reply triton/build_engine.sh:3]: OK, I didn't know the deprecation warning. That makes sense.

**Pull request merged**